docker-compose.yml

```diff
@@ -18,12 +18,10 @@ services:
     ports:
       - "9090:9000"
   localstack:
-    image: localstack/localstack:0.10.8
+    image: localstack/localstack:latest
```

Comment:
Should we lock this to a specific version?

Reply:
(I'd be worried about it changing underneath us)

Reply:
👍 I guess it's like any other dependency.